docs/source/wait_for_user.rst

```diff
@@ -43,7 +43,7 @@ That's it!  When you reload your app, you'll see a white page for a moment
 and then everything will render with the proper user state.
 
 If you have a complex bootstrap process, and want to show some kind of loading
-overlay while the application is bootstrapping, you may find the
+overlay while the application is bootstrapping. You may find the
```

Comment:
@timothyej 
Yeah, I didn't catch that one. That's a mistake. I fixed it. Thanks.